test/scripts/filters/meta_generator.js

```diff
@@ -32,4 +33,22 @@ describe('Meta Generator', () => {
     const resultType = typeof result;
     resultType.should.eql('undefined');
   });
+
+  it('ignore empty head tag', () => {
+    const content = '<head></head>'
+      + '<head><link></head>'
+      + '<head><link></head>'
```

Comment:
Looks like there are two tests here; one is "ignore empty head tag", the other is "add to the first non-empty head tag". Suggest to split them up.

Reply:
'apply to first non-empty head tag only' is now separated from 'ignore empty head tag'.